### ISSUE TITLE
Stop automatically exposing details relation for figure -> figcaption

### ISSUE THREAD

**Issue by aleventhal** [w3c/html-aam#564]
I thought we had filed this, so there may be a dupe somewhere.
We should stop automatically exposing the details relation between figure and figcaption. They are nearby in the DOM anyway. It was so noisy on some pages that it caused JAWS to stop reading details relation entirely. 

+ @scottaohara

**Comment by pkra** [comment]
> I thought we had filed this, so there may be a dupe somewhere. 

@aleventhal  maybe you're thinking of https://github.com/w3c/aria/pull/2224 ?

**Comment by aleventhal** [comment]
Thanks @pkra . I mentioned it in a comment in that PR. But it hasn't been changed in that PR. Should we keep it as a separate issue? Or do you want to add to that PR?

**Comment by scottaohara** [comment]
<s>i was planning on handling it in the PR - i've just not had the time.  someone else can update the PR, otherwise it's on me to get it done - and i will as soon as i can.</s>

i have updated the PR

**Comment by scottaohara** [comment]
@aleventhal to be fair though, since the PR never landed because we've been waiting on other browser to implement, chromium should be fine to remove it even without the PR update.

**Comment by scottaohara** [comment]
let's keep this issue open so it can come up in triage - and then this can be a reminder for people to re-review the PR so we can get it merged asap

**Comment by spectranaut** [comment]
Discussed in today's ARIA working group meeting: https://www.w3.org/2024/11/14-aria-minutes.html#d8ca

**Comment by aleventhal** [comment]
FWIW Chromium stopped doing this in milestone 131, to match other browsers. So we just need to make it official in the spec.
